app/controllers/api/v2/job_invocations_controller.rb

```diff
@@ -137,6 +137,24 @@ def rerun
         end
       end
 
+      api :GET, '/job_invocation/:id/outputs', N_('Get outputs of hosts in a job')
+      param :id, :identifier, :required => true
+      param :search_query, :identifier, :required => false
+      param :since, String, :required => false
+      param :raw, String, :required => false
+      def outputs
+        hosts = @job_invocation.targeting.hosts.authorized(:view_hosts, Host)
+        hosts = hosts.search_for(params['search_query']) if params['search_query']
+        raw = ActiveRecord::Type::Boolean.new.cast params['raw']
+        default_value = raw ? '' : []
+        outputs = hosts.map do |host|
+          host_output(@job_invocation, host, :default => default_value, :since => params['since'], :raw => raw)
+            .merge(id: host.id)
```

Comment:
Sorry for not noticing sooner, but while testing it, I've realized `outputs: [{...., id: 1}]` is quite missleading. First sight it seems like an ID of the output. Should we use `host_id` instead to be more explicit?
```suggestion
            .merge(host_id: host.id)
```